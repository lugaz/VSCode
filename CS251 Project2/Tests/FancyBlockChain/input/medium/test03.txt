442 array
4 -59.179970 1225
RlS5EThEp -75.905180 2940
lzyn -57.573964 2140
h79ivPW 76.359389 4454
6IHYaRm 24.598247 4516
Og -121.702246 2895
Jti -46.792406 4995
LnP -58.128970 3479
V3 -97.319458 3452
GIy5T2 -27.098428 3755
VC -70.409416 2179
XcL9HXLz 46.963437 2912
zyGD -166.602263 1660
jWJ 53.791096 3316
n5 110.299134 2900
qurQS -58.672178 1149
WduUV39U1RzG 152.090076 3718
6P5dGKF 58.596325 2238
VV32gpVnj -85.928003 4998
bVlaBKrrXa -19.563143 3620
09UgR -27.406609 3325
dN3KylIqRbAFl 30.045243 2812
srSRkUF 131.741240 4027
wpqZT3 -91.481389 1619
ovnDdYZ -36.093630 3363
rw -38.549880 2346
WFRmnBlW4u 35.040566 3167
2EbYC7J8S -159.452267 3296
BqmJvlC -36.231774 1714
Ktl -79.386664 1408
TPB7nmdfPm8Y 83.458890 3379
9kxA7Wo0Rna 22.419889 3655
CxGKYcsGM -51.743790 1979
HS4j2hoZPG -34.228561 3556
f 166.010387 4923
OkYT8ACgK 67.463481 1305
v4 96.280422 1219
bHOtdAuGU -31.690390 1383
4KOq9YK6jyRlX -19.537687 3128
xuiiITE4PEV 141.583673 3750
MzzUlY9rZD 60.608577 2146
yvN -86.192710 1732
SYXjxeMjbgZGo 104.483656 3399
rQsm1UTUa 5.826843 3781
Ka -222.469238 4500
zd3C7eC 32.759664 2062
jtI -73.672151 3935
MgXQkUk5 73.224889 2990
e9HPZhSg0j6 -142.150380 3261
H69UaXFK 35.368046 2814
8gTV5X -239.310491 2543
wNjRbGbuFW -168.606973 4015
LlMkFQ6h5qXk 88.637993 2516
vP1woGg -77.748691 1565
aEk2q6d4MXXI 3.896355 2031
h1 152.846566 1756
LI 76.132402 4527
r1u5svx9 15.186478 3295
r5ks8MU2Myz -18.449780 3210
Tm2Y 109.512644 1078
OwTUeO -66.671438 2145
7C43bDQte -133.467316 3912
Jzr 11.457410 3601
OHqnDCoQ1CaB -74.712793 4100
7sX8PrYdcnB -138.861701 1449
rp70RCwv3AsH -101.596438 3432
RarlraQejPkVE 214.799665 2935
L -58.004510 4900
NgtrzN7uh -27.034992 3393
fLOhj9 -35.141962 4059
xe 1.422933 1678
gp8JMNNu0X -107.253819 4474
PN9LM73c9no 213.729384 4644
w -106.045836 2984
4i7ei 241.452758 4696
GhpimEqz -280.213204 1166
3wO1HgvT -197.979100 4300
KanTWYmqHlf8 -74.817411 3411
0ZREhbSjRzY 138.867681 1090
Us28r 54.044965 2420
QMdhosb -79.305872 2102
4T 51.194222 2371
WMV7ub8EkmQ -38.549504 2080
Q -52.488774 2643
P6RlAjD1C1iwh 44.792100 4662
ZcO3 113.864000 2504
0dRM3jKxW6yf6 80.960219 3224
vu92Pxfdr4 -77.820052 3041
iIFrqIWfjvioC -77.372382 3450
O 146.098012 2591
BTLN -12.535187 2375
XIfpq -122.733001 2427
E65Jy 31.735672 4079
RSp6 115.555123 1182
bpTLAAvQc -19.442389 3346
gbaqy2hxmA4e -103.084359 2878
G -14.932085 1963
ZEN2ZNi 95.753827 2181
TmSQYdg4OIM -122.649249 2173
IxxgEsVv6T9b -8.928342 2071
64CbladKTJ -90.664176 2288
9d1ILRrpkVL -16.221744 1041
O8dAIcG -109.417678 2832
tT 77.594894 2972
u 11.716668 4040
MJ 4.711791 4654
9SUmbTy -20.127776 4095
Jz 237.275667 1133
gQ8ZHdRZNRM -41.889860 1414
9HBxLbmxC 51.957261 3334
aWi2QyL0Fyg 76.020770 1297
ubN6mgtcCjaJ -13.379208 3324
2JQ1D1Ofky 69.931582 3323
XOWIESN 46.246052 4151
spvkztTPKi -25.250121 3770
2p42ve3WEsh -42.232240 2545
yBoXpo6Dov -107.326387 3885
V2yMDMF6mWfi -59.069617 2358
7JW9IxalnKdg 196.231774 1376
RPQoc2 -107.379308 2426
CXvUH0sm8b 119.783370 4129
n97KxvPFK -157.185809 1179
TwgHdgAAUb 48.108623 1430
m9yfIJDBh 51.261218 3801
OnTe -156.604500 3147
qgQLS2l 5.924101 2170
diTo2Kg60Jm2s -147.178433 1311
TZnsKMRB 184.317258 3670
RM6q2lzu 13.480253 4349
axzh1 -18.723640 4119
WcUplYl2rt 74.060240 4905
HVQiTGxjDD 121.220269 3853
T8w 68.837891 2661
3EOToOTRZ 161.388248 2130
5qZluFbSNDz -55.251457 2274
UjBIOv3X -60.706804 2867
kaG2 -63.468335 2579
ULGAfP1Xna -9.279553 1894
d9 -108.342164 3747
9gigplIAN -111.630104 1976
7qdRUqdl531O 12.166257 2291
xZuGRkLEWv -34.449025 4184
jP 88.571247 4375
R -55.488368 4366
dGeQQ7xLs 0.733175 4416
EsCnXeQNM -103.157836 4145
in 57.167301 3347
E1rs7 -199.225855 1070
25LPNcGhlx -37.436017 3880
tj -21.159965 4548
Una -136.323797 1312
q3KzBwCy6 -122.445322 2921
iSymCd 72.378836 2165
UsqqcQKWmyDoI -25.321864 4778
T -45.052014 4215
oq2iJV -108.835265 3321
v -279.336646 2405
7c5Za5Fhjj5 5.903767 4855
cOG -20.288462 2276
r2VZ 31.216078 2227
tXaHaN 0.724654 2977
NN3nR2z4LhU 122.200749 1709
wSBGinfRgMCN1 -209.045065 2729
CXeHb -113.746432 1045
G15ngcQ 112.677544 1816
alFZ -186.533625 3706
EqRNEMtB9SMt5 -117.962341 2482
7T9U 28.983486 3353
bKz -64.495939 1432
x 4.659242 1508
FqoZGXuwl 89.360495 4735
STbf 80.469387 1672
8yxrDZ -35.222915 4236
you9pV9SZB -148.903869 1037
kgkKh8VCli -118.275198 3837
b -0.126510 2464
QvmatenJ 52.116339 2951
M 21.307122 1830
dlNuq 48.378549 3052
Nb6vQ 153.604143 1587
YEvxe7L -59.456802 2370
ZxrGKluc5petJ -196.195564 3724
g6F4FwmKAmoP -131.598832 4103
LS8lpD1nc1LR7 28.082339 3822
Om2PMxp 12.978750 2675
m -45.614311 4739
SXG 117.072273 3065
jmKRU -32.561622 1719
W4brfbTJTMKwH 129.967553 3838
2aT0 -142.123459 1303
0 74.696767 2876
XaWerlIgBm2 -117.388459 2285
sz4G 0.159341 4188
tS7McZ8A -34.084688 4308
AlD1MGrS -79.401664 1826
SUi -40.650099 4718
9bq -40.253666 1589
9eWDnBz -270.399340 1280
q -40.538702 4947
gmlWvfbl -83.590156 3666
Ke 12.583451 4143
ohSsxhf4u -121.801781 3650
bmmUw 45.694959 3212
Gay9srEjT -128.173323 4348
cMzEEaIm 49.159341 1579
hGRrDjlEoh -43.522482 2772
LiOOJvmtB2AAP 30.675178 3887
k632 -3.451691 1362
YICT63pQ -99.497076 2469
5y58b -31.185228 1054
Unk 55.860919 2725
Ri7CO -36.065149 1343
LH2nUUY -10.315750 2562
VsYCJZXlHmX 33.386276 3771
Wr8lWkGHY8eb -82.325139 3625
ZH6yqXblsHXr 160.062199 2161
NAK8ewd61HQxB -34.498281 4874
hCPwDVw -214.642870 3850
PASeQP 5.471749 1267
cvOYKTksoTij 3.904394 1367
aGLCVNUWb25 33.967975 4578
QnB -34.134777 4490
kF 167.274741 3571
gMg8qZ1ynRDe 80.338887 4712
A 44.623403 4131
yB -119.265977 3187
4sE59DOyKX -46.924446 2104
SI3KPhw6 145.874703 2349
c3Fpxn 43.153347 1081
hyyN -43.668408 2488
gz2AmUlde 183.299625 3445
V1Bcxzk6fO -176.370695 3800
j 125.348683 1753
9OH99 -82.720859 1067
S8ijXj 117.818824 1950
yNDSED 99.383684 4133
0Kx2qdEy1y -241.856233 3062
OyGPg8O -11.869564 4231
eETscMTCgUrLT 13.533394 1130
l -62.522915 4643
b038EcOWnT -104.854157 2024
Fb4I0F5v -212.980163 1244
FsDGxk4 -108.525811 2460
05u -13.273592 2383
4vzt1c -9.353818 1250
JasSFG 128.689494 2476
snzd38vALTJ -94.716529 4978
XiRI5H1jix 119.160987 3754
f7PLJdBD0 89.345142 4368
C2JJBI0 11.442263 2129
OfIZ -96.306307 2037
aon8huxJP1MGw -51.141006 3681
25 140.342180 4312
9sV -70.574595 2045
eVxj 57.165297 1623
QH7d4eDI7G -156.417692 3359
qwMnlY -73.618588 4345
pj25ZDCY2a 63.404303 1982
uUQy -118.749767 3570
ATTfC 37.574490 3225
AAHd 72.756679 4989
WM9mSKNj -22.955932 1543
z7cAdYi3TL -59.311923 1237
AE0r -55.598316 4315
BcCdzfKeJLB 84.846415 4022
ujgss 103.449710 3660
qo9ai9A7sf5I -76.158458 2926
wuiVgdeDGzSTm 8.216283 1001
4KEi4b4lZbv 51.485702 4503
QyuzwyMHqM2k 86.748245 3370
DNyt -243.480423 4393
sxocMyNR -157.624299 4656
4OqfjHYGpxV 74.712724 4642
6HY 81.762281 4350
Zr 10.926026 1775
FYxQhxcT9San 86.877214 2509
wLyaOi3Ed7V 2.843461 1079
StMMgNW 44.552312 1021
qywZKq0w8OTIE -75.678400 1424
wAp6N0n 97.873358 3533
JNmQ2pR4 32.002568 4992
bzflnJaVqOZjD 30.217931 1510
Jq4 90.857995 1990
eZjoKrkfUJt -23.695324 4963
0ATJpqXomU0f -54.933882 1644
bojdOomRZVgp5 -9.225347 2925
FPfs 46.396500 1690
CMIDCISLCMiWS -9.076368 3473
vRtWigi 47.240774 1386
iUTSQs -23.167871 1275
G4oBQ 85.247728 1304
ylNM65AlEdWf 8.167062 3989
6BAg8YOVs -166.125450 2437
cYRn3d6LcN -68.786267 4209
lJzupF7Gpx -219.738378 4532
D8vVPLnTSl -228.063881 1975
LiJhB0ud9gE 211.219331 1518
kyrwzaAFEX -249.760136 3992
SLwpSmFHt45h -37.188978 3793
kEX0qfgy 277.261950 3007
6ubfvcJ0 -159.395384 3746
mJfA 164.562914 3841
mg1uD -56.605396 3594
WEgLBGvf9n3 -138.091837 3354
IRVKyYZlJw 6.974843 3944
k6dDel 41.437906 2457
Qo6mqaC17GPH -6.058171 1904
ov 115.781049 3804
wE -6.581444 3758
ydo9 -49.572702 4233
7lrY0 -30.548477 4333
fTTVsAJzo 103.324412 2230
a69OHm7L 12.839413 3298
khbD 55.912830 4776
UtJeYsdo 111.961311 2144
L6XafheZHV -81.212737 4904
oGX0hD03xUTTY 271.326634 2673
1 102.367066 3251
qKH1XkZf 202.803034 3075
379rfRHa 1.479104 2310
V2ez -74.659156 3181
Od3DPC -39.379489 3672
Fc7d83j 48.799510 4538
Rb7gRWV6X64W -150.742139 1896
zk05f 14.325783 2863
9rVy5S -28.795673 3140
COcOFb0OipsXj -22.342028 1260
bVzF -26.882732 3820
b4p8DlwcJ146 -134.248904 4099
gXEw2ixGwwneX 84.130653 3327
3k 160.298507 1385
vI5Cjq5aRXFVh 127.174128 4404
1ZrCm -136.530306 2742
477fanKwY 110.970952 1233
4kx5F1 105.858168 4303
C -66.935674 4139
Bi7uF -196.556803 1988
zn7g1U -171.468540 2474
cTwAfr 73.252266 3303
I9Kuaw6bgkoKc 56.616856 3954
iT9AIVJjBx -9.189221 1712
7H -115.096803 4194
tUYlWqduz 126.509419 4254
xcPdRauB53 138.735586 4831
R4hS1l2E5Vb -195.629464 3704
8v -18.793158 4897
MgwK41v 59.106211 1338
jkdktF -36.436375 1494
3Dj7F3 -144.977732 2592
NcwwVv1vc7gWZ 211.404455 2060
zAFZDT25Onm -14.977739 4929
gcI -159.100029 1857
dE9qMpH 135.792061 4154
TIfy -112.177679 2662
IGvW3gJozs3NP 67.781290 4803
3wBaR5c -86.118258 4636
qUh6vGivdLg 1.735966 3744
uKErX3OXq 3.999522 3639
lE86 -83.387225 4990
tlanS -140.978979 1558
kRjrO7F9 -90.100067 4698
6g -62.863160 2054
2KFr -48.999811 3751
jlb0dM0cFlxo 105.928874 2402
JnbJI4 37.405919 2575
IrLSoF -75.181118 4023
LIcgfuCEpBnbi -95.430446 3390
TZ7mkrb9x -171.432515 2598
Ds9sPx 50.503879 2932
jF -33.558242 2599
Wu 48.627740 2189
CWwaWl2cHQ5 13.969749 4450
gBjAu5sOKk 120.931676 2120
kF7W 108.161821 1120
BdlO 36.221497 4547
4i0wV9hzA 111.768308 2657
8 3.536722 4243
sLvYlzrE 75.223660 2312
oozLp 54.495677 3292
iP 106.913503 4456
eMwN -144.255683 3417
4A4 -16.433034 1534
7FmXJYtQ 40.334744 2418
K -72.295939 2128
3RoHHY 11.665194 1274
3wfmciBk 153.958386 4054
L5FD -18.297204 4011
NgCloX8JuN -32.897319 4012
IRXH1mh 95.068591 1943
RsF4 -103.757672 2126
XQ0pX7YasuYPt 223.690665 4646
28mlE6wXo -100.684249 4498
ClEAX3 -111.361081 3384
B3 -159.141657 4033
U64 -53.484164 3609
e -54.030131 4093
BcKN3iVFht 69.326666 2493
tx 123.532965 2021
BAszT -132.091755 1393
W6F 95.125902 2087
FT4jq0N -110.330077 3680
Pwz -29.054972 4469
ctjt 161.864799 3469
tkcXQ -127.135056 4121
y 99.023277 3875
Z5fJLqG7lBz -29.595695 1418
dMJClDHFd 65.228253 1702
tYE8h -82.948031 4445
MfOy 173.478823 1935
TfCx2oAufQ3Ry 151.371064 4280
wSGfHUdoB -12.329195 4863
eJcl25n8 84.359879 4483
Agynn2BIDHvp -131.199773 2314
wcU5WDJ3csY 113.791390 4847
DJR2 287.522989 1669
mZu5WUJrtb8 127.440840 2212
vwGZTu5 -123.563320 3920
Fo3unsvLyEkp 163.498882 1025
FKAK -41.095658 1173
F 59.681023 4220
W 40.699684 3961
Uudj8DASJdK8 -48.249993 4107
iwBS 210.699807 1852
qTRZ8Zo -191.523163 2653
bYBlNUlpFU -61.897605 3068
nO9iJ7U6k -11.949144 2399
PFjbXLc -101.031668 4826
EjmWY8Bp0D 130.718831 1689
PZjSqypiWjgF 54.938412 4705
R5r2SN -125.860500 1765
QEHZFj 17.868934 2406
8CQJOe4iT 44.638393 3170
4vLbfMByKaHx -156.942579 3510
Ut3MvOJcMloj -44.406884 2969
RCaZp 149.529615 1193
CxedxaoNwhUOQ 156.585407 4406
ZCWHytEDj -24.955413 4074
w3nZdkzT5 21.113093 3391
yzO -55.606513 1154
hwy6i 235.299681 4305
Ga2DKhB6q51cy 106.908648 1757
AsdBxrbqHlR -94.740544 4976
updateEarliestBlock -13.859963
updateBlock CxedxaoNwhUOQ -50.353062
getEarliestBlock
addBlock s 1.712123 1483
updateBlock apUTCh -152.719657
getEarliestBlock
removeEarliestBlock
removeEarliestBlock
addBlock 4dQ 82.999982 1939
removeEarliestBlock
addBlock W3tV 10.538637 4161
addBlock rMHJU2w5nmb 22.980648 965
getEarliestBlock
addBlock HO6NAnYyqR -8.183800 838
updateBlock OHqnDCoQ1CaB 72.767292
addBlock oBen -150.053892 2630
removeEarliestBlock
removeBlock EsCnXeQNM
addBlock maJbHZcexEBn -3.188421 3959
updateBlock lkY2 -102.158962
removeEarliestBlock
addBlock vfiFbVfnNw9S -108.990775 2870
addBlock tp4WaS0 104.924793 71
updateBlock aGLCVNUWb25 42.795867
addBlock WSAtlcvnXoLE -59.186070 606
removeBlock c3Fpxn
removeEarliestBlock
updateBlock G -132.171887
removeEarliestBlock
updateEarliestBlock -202.747315
getBlock cYRn3d6LcN
getBlock R
getEarliestBlock
removeEarliestBlock
addBlock KHzRV39tAIFND 165.059087 1396
updateEarliestBlock -8.297629
getEarliestBlock
getBlock Z5fJLqG7lBz
addBlock c3Fpxn -65.052311 961
updateEarliestBlock -35.412231
getBlock wspNmhl
removeBlock mZu5WUJrtb8
addBlock nSFXHMA9lto 13.759029 2507
removeEarliestBlock
getBlock 9HBxLbmxC
addBlock cE3fZniHypPnX -238.407998 261
updateBlock lDM 109.717190
updateBlock MgwK41v -126.918755
removeEarliestBlock
getBlock XOWIESN
getEarliestBlock
addBlock BfLK 96.068440 1177
updateBlock LIcgfuCEpBnbi 69.067184
addBlock 5jpzn -61.250897 661
getEarliestBlock
removeEarliestBlock
updateBlock u -86.382752
removeBlock vu92Pxfdr4
addBlock lVH -62.551220 3826
removeEarliestBlock
getBlock lVH
getEarliestBlock
getBlock dE9qMpH
removeEarliestBlock
getBlock ZCWHytEDj
removeBlock iwBS
getBlock VV32gpVnj
getBlock XOWIESN
getBlock 9d1ILRrpkVL
addBlock 0W 124.465254 1379
addBlock q3eH 30.383903 3235
removeBlock alFZ
removeBlock D74BUEsB
getBlock LIcgfuCEpBnbi
updateBlock ujgss 17.499138
updateEarliestBlock 70.177408
updateBlock dE9qMpH 73.493021
getEarliestBlock
getEarliestBlock
removeEarliestBlock
updateBlock Z5fJLqG7lBz 113.743029
updateEarliestBlock 83.018053
updateBlock you9pV9SZB 17.327654
getEarliestBlock
getBlock I9Kuaw6bgkoKc
removeBlock 4kx5F1
removeEarliestBlock
removeEarliestBlock
updateBlock 3wBaR5c -23.145763
updateEarliestBlock -130.256227
removeBlock SLwpSmFHt45h
getBlock 9eWDnBz
updateEarliestBlock 5.841829
addBlock 3Z 125.944397 3418
getBlock bVzF
updateEarliestBlock -19.140651
getEarliestBlock
updateEarliestBlock -32.401413
removeBlock 8CQJOe4iT
addBlock kscLdt0u1yWn -65.250412 1323
getEarliestBlock
addBlock vNG18wuMpyu0 5.189319 801
updateBlock L5FD -111.789784
addBlock wF 154.147173 2606
getEarliestBlock
addBlock Oe -23.751527 4935
getEarliestBlock
removeBlock Wr8lWkGHY8eb
removeEarliestBlock
addBlock SFPgj 42.457316 1845
getBlock jkdktF
addBlock 8th -84.098920 830
removeBlock 4KOq9YK6jyRlX
updateBlock jP 11.875981
updateBlock K -97.003907
removeEarliestBlock
addBlock gBgtsQ1sfE2O 86.926264 3840
updateBlock PZjSqypiWjgF -24.861023
updateBlock QH7d4eDI7G -48.104161
getEarliestBlock
addBlock TZj4GY6Xz0N -68.803121 867
getBlock wE
updateEarliestBlock 33.888232
getBlock qKH1XkZf
addBlock SLwpSmFHt45h 46.546864 1798
updateBlock tYE8h 80.812267
updateBlock CXeHb -63.453484
addBlock HYQj8I -126.870734 1796
updateBlock 3Dj7F3 33.372334
removeBlock rw
removeBlock rMHJU2w5nmb
addBlock bpZIl6L -86.280377 3561
removeBlock R4hS1l2E5Vb
removeBlock tlanS
getBlock 7lrY0
removeBlock wAp6N0n
getBlock aEk2q6d4MXXI
removeEarliestBlock
updateEarliestBlock 5.358895
removeBlock p8LmxuXM8
getBlock FYxQhxcT9San
updateEarliestBlock 131.714479
updateBlock 0ZREhbSjRzY 156.987640
updateBlock e0LBI0Gk -89.326159
removeEarliestBlock
removeEarliestBlock
addBlock 4TUN5MClHJ -88.001394 668
updateEarliestBlock 147.719680
updateBlock gMg8qZ1ynRDe 144.698758
getBlock OnTe
getBlock a69OHm7L
getEarliestBlock
removeBlock FT4jq0N
removeEarliestBlock
getBlock yzO
updateEarliestBlock -80.569376
removeEarliestBlock
addBlock 7G2TH0zGn5 -66.037539 394
updateEarliestBlock 214.471875
