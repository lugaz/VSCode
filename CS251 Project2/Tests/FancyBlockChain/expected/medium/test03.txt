StMMgNW 44.552312 1021
true
Fo3unsvLyEkp 163.498882 1025
Fo3unsvLyEkp 163.498882 1025
you9pV9SZB -148.903869 1037
true
9d1ILRrpkVL -16.221744 1041
true
true
rMHJU2w5nmb 22.980648 965
false
true
CXeHb -113.746432 1045
EsCnXeQNM -103.157836 4145
true
5y58b -31.185228 1054
true
true
true
c3Fpxn 43.153347 1081
WSAtlcvnXoLE -59.18607 606
9OH99 -82.720859 1067
cYRn3d6LcN -68.786267 4209
R -55.488368 4366
Tm2Y 109.512644 1078
Tm2Y 109.512644 1078
true
0ZREhbSjRzY 138.867681 1090
Z5fJLqG7lBz -29.595695 1418
true
null
mZu5WUJrtb8 127.44084 2212
true
0ZREhbSjRzY 138.867681 1090
9HBxLbmxC 51.957261 3334
true
cE3fZniHypPnX -238.407998 261
XOWIESN 46.246052 4151
kF7W 108.161821 1120
true
true
5jpzn -61.250897 661
5jpzn -61.250897 661
vu92Pxfdr4 -77.820052 3041
true
kF7W 108.161821 1120
lVH -62.55122 3826
eETscMTCgUrLT 13.533394 1130
dE9qMpH 135.792061 4154
eETscMTCgUrLT 13.533394 1130
ZCWHytEDj -24.955413 4074
iwBS 210.699807 1852
VV32gpVnj -85.928003 4998
XOWIESN 46.246052 4151
null
true
true
alFZ -186.533625 3706
null
LIcgfuCEpBnbi 69.067184 5008
qurQS -58.672178 1149
qurQS -58.672178 1149
qurQS -58.672178 1149
GhpimEqz -280.213204 1166
I9Kuaw6bgkoKc 56.616856 3954
4kx5F1 105.858168 4303
GhpimEqz -280.213204 1166
FKAK -41.095658 1173
SLwpSmFHt45h -37.188978 3793
9eWDnBz -270.39934 1280
true
bVzF -26.882732 3820
RCaZp 149.529615 1193
8CQJOe4iT 44.638393 3170
true
v4 96.280422 1219
true
true
vNG18wuMpyu0 5.189319 801
true
vNG18wuMpyu0 5.189319 801
Wr8lWkGHY8eb -82.325139 3625
vNG18wuMpyu0 5.189319 801
true
jkdktF -36.436375 1494
true
4KOq9YK6jyRlX -19.537687 3128
8th -84.09892 830
true
v4 96.280422 1219
true
wE -6.581444 3758
qKH1XkZf 202.803034 3075
true
true
rw -38.54988 2346
null
true
R4hS1l2E5Vb -195.629464 3704
tlanS -140.978979 1558
7lrY0 -30.548477 4333
wAp6N0n 97.873358 3533
aEk2q6d4MXXI 3.896355 2031
v4 96.280422 1219
null
FYxQhxcT9San 86.877214 2509
z7cAdYi3TL -59.311923 1237
Fb4I0F5v -212.980163 1244
true
OnTe -156.6045 3147
a69OHm7L 12.839413 3298
4vzt1c -9.353818 1250
FT4jq0N -110.330077 3680
4vzt1c -9.353818 1250
yzO 83.018053 5014
PASeQP 5.471749 1267
true
